Documentation/toc.yml

```diff
@@ -204,35 +204,49 @@
         href: Tools/ControllerMappingTool.md 
   - name: UX Building Blocks
     items:
-    - name: Interactable
-      href: README_Interactable.md
     - name: Button
       href: README_Button.md
     - name: Bounding Box
       href: README_BoundingBox.md
-    - name: Object Manipulation
-      href: README_ManipulationHandler.md
```

Comment:
Do we want to leave this in our toc, in case anybody hasn't updated for whatever reason? We could add an `[Obsolete]` tag to it like your `[Experimental]` tags below

Reply:
That's a great idea! I'll make the changes.